WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/CardPresentPaymentsOnboardingUseCase.swift

```diff
@@ -128,26 +137,59 @@ private extension CardPresentPaymentsOnboardingUseCase {
             return .countryNotSupported(countryCode: countryCode)
         }
 
+        let wcPay = getWCPayPlugin()
+        guard stripeGatewayIPPEnabled == true else {
+            return wcPayOnlyOnboardingState(plugin: wcPay)
+        }
+        let stripe = getStripePlugin()
+
+        // If both the Stripe plugin and WCPay are installed and activated, the user needs
+        // to deactivate one: pdfdoF-fW-p2#comment-683
+        if bothPluginsInstalledAndActive(wcPay: wcPay, stripe: stripe) {
+            return .selectPlugin
+        }
+
+        // If only the Stripe extension is active, skip to checking plugin version
+        if let stripe = stripe,
+            onlyStripeIsActive(wcPay: wcPay, stripe: stripe) {
+            return stripeGatewayOnlyOnboardingState(plugin: stripe)
+        } else {
+            return wcPayOnlyOnboardingState(plugin: wcPay)
+        }
```

Comment:
nit: might be worth adding test cases for this logic? 🙂

Reply:
Agreed! Opened https://github.com/woocommerce/woocommerce-ios/issues/5866